package.json

```diff
@@ -6,6 +6,9 @@
   "bun": {
     "autorouting": "./dist/cli.js"
   },
+  "bin": {
+      "autorouting": "./dist/cli.js"
+  },
```

Comment:
lol i think the "bun" thing above this might have been a typo.... we can probably remove it